bundles/automation/org.eclipse.smarthome.automation.commands/src/main/java/org/eclipse/smarthome/automation/internal/commands/AutomationCommandsPluggable.java

```diff
@@ -408,8 +412,8 @@ protected String buildCommandUsage(final String syntax, final String description
 
     @Override
     public RuleStatus getRuleStatus(String ruleUID) {
-        if (callback != null) {
-            RuleStatusInfo rsi = callback.getStatusInfo(ruleUID);
+        if (ruleManager != null) {
```

Comment:
The rule manager is a mandatory reference, so it cannot be null if the component is active.